### PR TITLE
Add get_note method (#849)

### PR DESCRIPTION
**Pull request by water-ghosts**
Adds the worksheet.get_note() method requested in #849, with relevant test cases.

### CODE REVIEW

File: gspread/models.py [hunk 1]
```diff
@@ -149,6 +149,22 @@ def batch_update(self, body):
 
         return r.json()
 
+    def notes_get(self, range):
```
Comment (burnash) [inline gspread/models.py:152]: I'm not convinced that there is a need for this public method in the `Spreadsheet` class. It'd be great to have as minimal api surface area as possible and I actually regret that I added other convenience methods as public methods in `Spreadsheet`.

Let me know if you have a specific reason for this public method. If you don't, please consider moving this code to `Worksheet.get_note`.

**Review by burnash — CHANGES_REQUESTED** [review]
Perfect, thank you very much! Please see the comment for `Spreadsheet.notes_get`.

File: gspread/models.py (hunk 1 above)
Comment (water-ghosts) [reply gspread/models.py]: I just added that method for consistency with values_get(). I’ve removed it and transferred the logic to worksheet.get_note()

**Comment by burnash** [comment]
Great, thank you very much again!

**Pull request merged**